about.html

```diff
@@ -0,0 +1,182 @@
+            <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Dolor, saepe fugiat. Ratione repellendus, aliquam omnis obcaecati, aperiam neque temporibus, laborum in repudiandae harum pariatur molestias quam! Nesciunt eius saepe unde quaerat ratione, in dicta quae iure aut eum quis nemo? Aut pariatur sint quibusdam tempore illo a dolore quo </p>
+        </div>
+        <a class="brandLogo pb-4" href="index.html"><span class="text-warning">Tech</span>Con</a>
+    </section>
+
+    <!-- See previous events -->
+
+    <section class="bg-dark w-100">
+        <div class="container">
+            <div class="text-center text-white pt-4">
+                <h2>Previous webinars</h2>
+                <div class="guide_bar"></div>
+            </div>
+            <div class="row justify-content-center pb-5 gap-4">
+                <div class="col-md-4 bg-image ripple" data-mdb-ripple-color="light">
+                    <img src="./images/virtual-pic-1.jpg" class="w-100" />
+                    <a href="#!">
+                      <div id="mask" class="mask">
+                        <div class="d-flex justify-content-center align-items-center h-100">
+                          <h4 class="text-white text-center">2021<br>
+                            TechCon in 2021</h4>
+                        </div>
+                      </div>
+                      <div class="hover-overlay">
+                        <div id="mask-2" class="mask"></div>
+                      </div>
+                    </a>
+                  </div>
+                <div class="col-md-4 bg-image ripple" data-mdb-ripple-color="light">
+                    <img src="./images/virtual-pic-2.jpg" class="w-100" />
+                    <a href="#!">
+                      <div id="mask" class="mask">
+                        <div class="d-flex justify-content-center align-items-center h-100">
+                            <h4 class="text-white text-center mb-0">2022<br>
+                                TechCon in 2022</h4>
+                        </div>
+                      </div>
+                      <div class="hover-overlay">
+                        <div id="mask-2" class="mask"></div>
+                      </div>
+                    </a>
+                  </div>
+            </div>
+        </div>
+    </section>
+
+  <!-- Global sponsors -->
+
+  <section>
+    <h2 class="text-center text-dark mt-4">Partners</h2>
+    <div class="guide_bar"></div>
+    <ul class="sponsor-class">
+        <li><img class="col-md" id="sponsor-logo" src="./images/naver-logo.jpeg" alt="Naver logo"></li>
+        <li><img class="col-md" id="sponsor-logo" src="./images/daumkakao-logo.jpeg" alt="Daumkakao logo"></li>
+        <li><img class="col-md" id="sponsor-logo" src="./images/airbnb-logo.jpeg" alt="Airbnb logo"></li>
+        <li><img class="col-md" id="sponsor-logo" src="./images/mozilla-logo.jpeg" alt="Mozilla logo"></li>
+    </ul>
+
+</section>
```

Comment:
- [ ] As the previous reviewer mentioned, The sponsor section for the About page should appear only on the mobile version, So kindly work on that 👍
- [ ] Also, according to the original design the partners section should has a dark background-color 👍

> Desktop Version, there's no partners section

| Your Design  | Original Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/198779792-b55ed8e9-c635-425c-94c7-9fd9df72d91a.png" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/198779606-02f4a74d-e8d9-478c-806c-3e01ef9e1688.png" /> |


> Mobile Version, there's partner section with dark backgournd

| Your Design  | Original Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/198782290-b047ce89-99ff-4c77-a9ef-3ddf533c4371.png" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/198781858-5749a9bf-d7ed-49f7-b5f3-ed27a549dbd7.png" /> |